src/coreclr/src/jit/indirectcalltransformer.cpp

```diff
@@ -8,8 +8,8 @@
 #endif
 
 // The IndirectCallTransformer transforms indirect calls that involve fat function
-// pointers or guarded devirtualization candidates. These transformations introduce
-// control flow and so can't easily be done in the importer.
+// pointers, guarded devirtualization candidates, or runtime lookup with dynamic dictionary expansion feature.
```

Comment:
Nit: I would replace "dynamic dictionary expansion feature" -> "dynamic dictionary expansion" everywhere. We do not add feature anywhere else.

E.g. This comment does not say "fat function feature" or "guarded devirtualization candidates feature". It just says "fat function" or "guarded devirtualization candidates".